frontend/app/helpers/spree/frontend_helper.rb

```diff
@@ -71,14 +71,17 @@ def checkout_progress(numbers: false)
       content << content_tag('div', raw('<hr /><hr /><hr />'), class: "checkout-progress-steps-line state-#{@order.state}")
     end
 
-    def flash_messages(_opts = {})
+    def flash_messages(opts = {})
```

Comment:
@aplegatt please don't change it, this breaks rubocop rules

Reply:
Leaving it with name `_opts` gives me rubocop warning: `Do not use prefix '_' for a variable that is used. [Lint/UnderscorePrefixedVariableName]`